cwms-data-api/src/test/resources/cwms/cda/data/dto/watersupply/water_supply_accounting.json

```diff
@@ -0,0 +1,43 @@
+{
+  "contract-name": "Test Contract",
+  "water-user": {
+    "entity-name": "Test Entity",
+    "project-id": {
+      "name": "Test Location",
+      "office-id": "SPK"
+    },
+    "water-right": "Test Water Right"
+  },
+  "pump-accounting": [
```

Comment:
You have a lot of repeated data here w/in the pump accounting JSON. Accounting is a time series, and this portion of the data structure will need to scale.

Reply:
Rewrote for better scaling